pwndbg/commands/attachp.py

```diff
@@ -71,7 +72,32 @@ def attachp(target) -> None:
                 return
 
             if len(pids) > 1:
-                print(message.warn(f"Found pids: {', '.join(pids)} (use `attach <pid>`)"))
+                pidsData.append(
+                    "{:<8} {:<15} {:<51} {:<51}".format("PID", "USER", "COMMAND", "PROCESS TREE")
+                )
+                for pid in pids:
+                    pid = int(pid)
+
+                    user = check_output(["ps", "-o", "user=", "-p", str(pid)]).decode().strip()
+                    command = (
+                        check_output(["ps", "-o", "cmd=", "-p", str(pid)]).decode().strip()[:40]
+                    )
+                    process_tree = check_output(["pstree", "-p", str(pid)]).decode().strip()[:40]
```

Comment:
What if pstree is not installed? :P 

Ideally it would be best to read data from `/proc/$pid/status` and other files, so that we don't depend on `ps` and `pstree` :/

Reply:
yep, pstree probably also uses /proc